File: Service/Admin/RMA.Service.Admin.SecurityManager.Contracts/Entities/Permission.cs
namespace RMA.Service.Admin.SecurityManager.Contracts.Entities
{
    public class Permission
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public int? SecurityRank { get; set; }
        public bool IsActive { get; set; }
        public bool OverridesRolePermission { get; set; }
    }
}

File: Web/RMA-Web-Shell/projects/shared-models-lib/src/lib/security/permission.ts
/** @description The Permission data. */
export class Permission {
    id: number;
    name: string;
    securityRank:number;
    isActive : boolean;
    overridesRolePermission : boolean;

    constructor(id: number,name: string,securityRank:number){
        this.id=id;
        this.name=name;
        this.securityRank=securityRank;
    }
}
